integrations/android_studio.rst

```diff
@@ -14,153 +14,104 @@ that can be cross-compiled to Android could be used using the same procedure.
 
 We are going to start from the "Hello World" wizard application and then will add it the ``libpng`` C library:
 
-1. Create a new Android Studio project and include C++ support.
+1. Follow the :ref:`cross-build your libraries for Android<cross_building_android>` guide to create
+   a standalone toolchain and create a profile ``android_21_arm_clang`` for Android.
+   You can also use the NDK that the Android Studio installs.
 
+2. Create a new Android Studio project and include C++ support.
 
-|wizard1|
 
+ |wizard1|
 
-2 Select your API level and target.
 
+3. Select your API level and target, the arch and api level have to match with the standalone
+toolchain created in step 1.
 
-|wizard2|
 
+ |wizard2|
 
-3. Add an empty Activity and name it.
 
+4. Add an empty Activity and name it.
 
-|wizard3|
-|wizard4|
+ |wizard3|
 
+ |wizard4|
 
-4. Select the C++ Standard
 
+5. Select the C++ standard
 
-|wizard5|
+ |wizard5|
 
-
-5. Change to the `project view` and in the `app` folder create a ``conanfile.txt`` and ``conan_android_profile`` files with
+6. Change to the `project view` and in the `app` folder create a ``conanfile.txt`` with
 the following contents:
 
-|wizard7|
-
 
 **conanfile.txt**
 
 .. code-block:: text
 
     [requires]
-    libpng/1.6.23@lasote/testing
+    libpng/1.6.23@conan/stable
```

Comment:
But this still not in conan-center, right?

Reply:
You are right! Maybe we should add it

Reply:
better to wait. there is a pending inclusion request for libpng that seems to be more up to date, but the GH link is broken, so, I'll keep the transit reference